### PR TITLE
Fix heapster on raw CoreOS.

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

File: clusters/coreos/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+pushd $( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+godep go build -a 
+
+docker build -t vish/heapster-buddy-coreos:canary .
```
Comment (vmarmol) [inline clusters/coreos/build.sh]: nit: build to something local since vish is only you :)

**Comment by vmarmol** [comment]
LGTM, just that one nit.

File: clusters/coreos/build.sh (hunk 1 above)
Comment (vishh) [inline clusters/coreos/build.sh]: This is a public repo. Any suggestions for other repos? I can use kubernetes, if you think thats ok.

Comment (vmarmol) [inline clusters/coreos/build.sh]: just "heapster-buddy-coreos" which keeps it local. If you want to push it to a repo you can do:

```
$ docker tag heapster-buddy-coreos kubernetes/heapster-buddy-coreos
$ docker push kubernetes/heapster-buddy-coreos
```

Comment (vishh) [inline clusters/coreos/build.sh]: Done

**Comment by vmarmol** [comment]
Needs a rebase is seems.

**Comment by vishh** [comment]
Rebased to HEAD.

**Comment by vmarmol** [comment]
Thanks @vishh :) LGTM, waiting on green to merge

**Pull request merged**